source/styleEditor.js

```diff
@@ -1,3 +1,17 @@
 $(document).ready(function(){
-  // your code goes here.
+
+
+  function changeProperties(selector, property, value) {
+    $(selector).css(property, value);
+  }
+
+  $('#style_editor').submit(function() {
+    event.preventDefault();
+    let selector = $('#style_editor :input')[0].value
```

Comment:
you can also select this "selector" name field like so: `$('input[name="selector"]')` 

same with the property and value fields 👍